src/nlp/builder.py

```diff
@@ -160,8 +159,8 @@ def __init__(
                 os.rmdir(self._cache_dir)
 
     @property
-    def does_require_manual_download(self):
-        return hasattr(self, "MANUAL_DOWNLOAD_INSTRUCTIONS") and (self.MANUAL_DOWNLOAD_INSTRUCTIONS is not None)
+    def manual_download_instructions(self) -> Optional[str]:
```

Comment:
Maybe add some docs with example to explain what a user that wants to create a script has to write in this function (cf what's done for _split_generators for example) ?

Reply:
sounds good - will keep this for a future PR